doc/index.rst

```diff
@@ -54,14 +46,16 @@ is slightly modified:
 
     python setup.py install --user
```

Comment:
@jzuhone - OK that sounds not quite so troublesome since you are explicitly opting-in to use the `--user` installation in a non-standard place.  It is still important to be sure that nobody ever does a vanilla `--user` install for testing which goes into the standard location in `~/.local`.

An alternative to the --user strategy is just to do a more traditional `--prefix` install.  E.g.
```
export prefix=/data/acis/LoadReviews
export PYTHONPATH=${prefix}/lib/python2.7/site-packages
mkdir -p $PYTHONPATH  # just once
python setup.py install -prefix=$prefix
```
This is pretty similar in the end to what you are suggesting except that it never exposes the evil `--user` option at all and it is much harder to accidentally do the wrong thing.  With `--user` for testing, it will go to `~/.local` silently (and all testing will appear normal) if you happen to forget to set `PYTHONUSERBASE`.

But in any case, both ways will work just fine with care.